modules/server/services/ballerina-parser/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@kaviththiranga  Better to include License header

Reply:
Fixed